modules/admin_manual/pages/configuration/server/occ_commands/core_commands/file_commands.adoc

```diff
@@ -265,13 +265,17 @@ This command is useful before removing users.
 
 [width="100%",cols="35%,70%",]
 |===
-| `--path=[PATH]`        | selectively provide the path to transfer. +
+| `--path=[PATH]`                 | selectively provide the path to transfer. +
 For example --path="folder_name"
 | `-s, +
---accept-skipped-shares` | always confirm to continue in case of skipped shares.
+--accept-skipped-shares`          | always confirm to continue in case of skipped shares.
+| `--destination-use-user-folder` | if destination user never logged in + 
+and thus has no mounts transfer + 
+directly to top-level user folder.
```

Comment:
The first line is a if
followed by a statemet - ok
but the next line does not point to a solution or explanation (directly to top-level user folder).
The sentence looks "strange/confusing"

Well done, your changes render the following ✔️ 

![image](https://user-images.githubusercontent.com/3321281/132984335-59d4a054-bcde-4fdb-a18d-606d5d393d75.png)

Reply:
any suggestions how to improve? my docs were really engineer driven

Reply:
Yes, this is difficult to parse. A comma after "mounts" should clarify things. Then the share is transferred directly to the top-level user folder.

Reply:
@mmattel @EParzefall adjusted, can you check now ?